src/commands/music/play.ts

```diff
@@ -35,12 +37,20 @@ module.exports = {
             return await interaction.reply( 'Please make sure to join a channel before running this command.' )
         }
 
-        // If the bot is already playing music, return and notify
         const connection = getVoiceConnection( voiceChannel.guild.id )
         if ( connection ) {
             if ( connection.joinConfig.channelId == voiceChannel.channelId ) {
-                return await interaction.reply( 'I am already playing music in your channel.' )
+                const row = new MessageActionRow()
+                    .addComponents(
+                        new MessageButton()
+                            .setCustomId(`switch - ${station}`) // to retreive station on button interaction
```

Comment:
The custom ID has a maximum of 100 characters (see [Discord Developer Portal](https://discord.com/developers/docs/interactions/message-components#component-object-component-structure)) and since we're processing user input here, we should ensure that the custom ID doesn't end up being too long.

Reply:
Oh I see, do you have any suggestions on how else I can send the station name from the message to the button?

Reply:
> The custom ID has a maximum of 100 characters (see [Discord Developer Portal](https://discord.com/developers/docs/interactions/message-components#component-object-component-structure)) and since we're processing user input here, we should ensure that the custom ID doesn't end up being too long.

This is a good point, Windows allows the creation of folders with a name of ~250 characters. Maybe a limit could be put on the folder names because giving a radio station a name that is more than 100 characters long isn't that user friendly.

Reply:
Makes a lot of sense. So should I let this part be as is? And revert the paths below to their original form? (Which will happen automatically once I resolve merge conflicts and accept the incoming changes)

Reply:
> So should I let this part be as is?

I think that's fine. In my opinion adding a limit for the station name length should have it's own discussion and pull request...

Reply:
> > So should I let this part be as is?
> 
> I think that's fine. In my opinion adding a limit for the station name length should have it's own discussion and pull request...

Agreed, I will merge this PR. A new issue can be created regarding the name length of folders.